torchaudio/models/wavernn.py

```diff
@@ -0,0 +1,103 @@
+from typing import Optional
+
+from torch import Tensor
+from torch import nn
+
+__all__ = ["ResBlock", "MelResNet"]
+
+
+class ResBlock(nn.Module):
+    r"""This is a ResNet block layer. This layer is based on the paper "Deep Residual Learning
+    for Image Recognition". Kaiming He,  Xiangyu Zhang, Shaoqing Ren, Jian Sun. CVPR, 2016.
+    It is a block used in WaveRNN(https://github.com/G-Wang/WaveRNN-Pytorch).
```

Comment:
Let's refer to the paper that has the original implementation, here and below.

Reply:
It has been updated.